NanoVNASaver/Marker.py

```diff
@@ -188,9 +101,9 @@ def __init__(self, name, initialColor, frequency=""):
             "s21groupdelay": ("S21 Group Delay:", self.s21_group_delay_label),
         }
 
-        ################################################################################################################
+        ###############################################################
         # Marker control layout
-        ################################################################################################################
+        ###############################################################
```

Comment:
Line length is 120, not 80. These changes shouldn't keep popping up again and again... :-)

Reply:
I've lengthen the comments in NanoNVAsaver.py....